### ISSUE TITLE
A11y_VisualStudioCodeClient_Extension_Search_Status Messages: Screen reader is not announcing the error message when we are using the undefined variable in debug console.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#29104]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

--- GitHubTags:
-A11yMAS; -A11yTCS; -A11ySev2; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2023; -WCAG4.1.3; -NVDA; -JAWS;

--- Environment and OS details:
Visual Studio Code insider version 1.82.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992
NVDA Version: 2022.2.3

--- Repro Steps:

1. Open Visual Studio Code Insiders editor  
2. Press ctrl+shift+y to open debug console.
3. Tab till debug console editor.
4. Verify whether the screen reader is announcing the error message when we are using the undefined variable in debug console.

--- Actual Result:
Screen reader is not announcing the error message when we are using the undefined variable in debug console.

--- Expected Result:
Screen reader should announce the error message when we are using the undefined variable in debug console.

--- User Impact:
Screen readers users may not be able to identify what error is there in the debug console.

--- Attachment:
